### PR TITLE
Issue template for support

### PR DESCRIPTION
**Pull request by justinegeffen**
Creating an issue template for support

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- type: textarea
+  attributes:
+    label: Description of issue
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Customer impact
+    description: How has this impacted customers.
```
Comment (llewellyn-sl) [inline .github/ISSUE_TEMPLATE/support.yml:15]:
```suggestion
    description: How has this impacted customers?
```

File: .github/ISSUE_TEMPLATE/support.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    required: true
+- type: textarea
+  attributes:
+    label: Environment/installation requirements
+    description: Specific requirements?
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Documentation URLs including 3rd party docs
```
Comment (llewellyn-sl) [inline .github/ISSUE_TEMPLATE/support.yml:32]:
```suggestion
    label: Documentation URLs, including 3rd party docs
```

File: .github/ISSUE_TEMPLATE/support.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    description: What documentation have you consulted?      
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Documentation to be updated on our side
+    description: Please include section if possible (FAQ, installation, etc)
```
Comment (llewellyn-sl) [inline .github/ISSUE_TEMPLATE/support.yml:39]:
```suggestion
    description: Link to existing docs pages requiring updates, or suggest categories/sections to add/update.
```

**Review by llewellyn-sl — CHANGES_REQUESTED** [review]
Minor changes. LGTM otherwise!

**Comment by gwright99** [comment]
> @justinegeffen @llewellyn-sl @gwright99
> 
> I wonder if so many boxes introduces additional friction?

I wondered about that too but am willing to withhold judgment for now. It's not like we open 100 doc tickets per day, and the only people to use this form are internal users who are self-interested to get it right. 

IMO a bit of friction is fine just to get something out there quickly which fulfills most of everyone's needs.

**Review by llewellyn-sl — APPROVED** [review]

**Pull request merged**